ee-feature-pack/common/src/main/resources/modules/system/layers/base/io/opentelemetry/otlp/main/module.xml

```diff
@@ -21,13 +19,22 @@
   ~ Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
   ~ 02110-1301 USA, or see the FSF site: http://www.fsf.org.
   -->
-
-<module xmlns="urn:jboss:module:1.9" name="org.latencyutils">
+<module xmlns="urn:jboss:module:1.9" name="io.opentelemetry.otlp">
```

Comment:
just out of curiosity, why are we adding micrometer related dependencies and modules to the ee-standard if the micrometer extension is only available via galleon-pack?

Reply:
The OpenTelemetry modules are already a part of the ee-standard FP. This Micrometer change requires some additional OTLP support, so, rather that bifurcating that configuration, I've opted to keep all of the OTLP-related module definitions together.

Reply:
@bstansberry Does this need to be done for WF 28 code freeze?

Reply:
@jasondlee Yes. Not for the beta though.

Reply:
@bstansberry, So not this week, but very soon after, then? If so, I'll file a follow up JIRA upon confirmation. :)